docs/customization/model.rst

```diff
@@ -399,11 +449,20 @@ Which we strongly recommend over updating the schema.
     $ php bin/console doctrine:migrations:diff
     $ php bin/console doctrine:migrations:migrate
 
+.. warning::
+
+    Sometimes when something goes wrong and you get database-errors you must use ``doctrine:database:drop`` but **remember this command destroy you database !!!**
```

Comment:
When have you encountered an error that made you drop and re-create the database? And what exactly means that `something goes wrong`?

Reply:
If i override an entity in my new branch and try make migration then I get errors and one way to fix this is drop database or switch to master them make migration:migrate and back to my new branch, make diff:migration and migration:migrate

Reply:
IMO we should not recommend such a solution - people work not only locally but also using docker or vagrant, there are too many cases when something can go wrong (or go well) with database and our tricky quick-but-a-little-hacky solutions with dropping a database and creating it again does not have to work for everybody :)

Reply:
Summary you have a right, I added this message becouse I have been problems in my env, people that use our guides has more different envirnonments and they propably can get many cases can go wrong

Reply:
To me, it sounds like `If anything goes wrong, you were not cautious enough and you have to throw your app away and build a new one` :D